### PR TITLE
Small bikeshedding lint

### PR DESCRIPTION
**Pull request by tkrotoff**
Nothing fancy, discovered this while playing with the code

- no need to import with .js extension
- use destructuring
- use template literals

### CODE REVIEW

File: js/tests/unit/util/config.spec.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ describe('Config', () => {
       const obj = new DummyConfigClass()
       expect(() => {
         obj._typeCheckConfig(config)
-      }).toThrowError(TypeError, obj.constructor.NAME.toUpperCase() + ': Option "parent" provided type "number" but expected type "(string|element)".')
+      }).toThrowError(TypeError, `${obj.constructor.NAME.toUpperCase()}: Option "parent" provided type "number" but expected type "(string|element)".`)
```
Comment (WinterSilence) [inline js/tests/unit/util/config.spec.js:43]: redundant changes: template strings work slower and and don't give any pros in current case

**Comment by tkrotoff** [comment]
Any news?

**Comment by tkrotoff** [comment]
Any news?

**Comment by mdo** [comment]
@GeoSot @XhmikosR Thoughts on this?

File: js/src/util/focustrap.js [hunk 2]
```diff
@@ -85,10 +85,10 @@ class FocusTrap extends Config {
   }
 
   // Private
-  _handleFocusin(event) {
```
Comment (GeoSot) [inline js/src/util/focustrap.js:88]: is like this on purpose 0c3dfe104b520d6ce466f265bf60c6d74785972e

File: js/src/util/index.js [hunk 3]
```diff
@@ -88,8 +88,8 @@ const getTransitionDurationFromElement = element => {
   }
 
   // If multiple durations are defined, take the first
-  transitionDuration = transitionDuration.split(',')[0]
-  transitionDelay = transitionDelay.split(',')[0]
+  [transitionDuration] = transitionDuration.split(',');
```
Comment (GeoSot) [inline js/src/util/index.js:91]: the initial version is still easier for the eye ;)

**Review by GeoSot — CHANGES_REQUESTED** [review]

**Pull request closed without merging**